### DISCUSSION TITLE
Issues with SSL Configuration and Reverse Proxy Setup in Nextcloud AIO with BunkerWeb on Proxmox

### DISCUSSION THREAD

**Discussion by psychedelicu** [nextcloud/all-in-one#5097] [Questions] [answered]
I recently moved my Nextcloud All-In-One (AIO) installation from Nginx Proxy Manager (NPM) on Unraid to BunkerWeb on Proxmox. Since the move, I've encountered significant issues with SSL and reverse proxy configuration. Specifically, I'm getting an SSL error: net::ERR_CERT_AUTHORITY_INVALID when trying to access my Nextcloud instance via the domain.

I've configured the trusted_proxies in config.php to include 127.0.0.1, ::1, and the Docker subnet (172.19.0.0/16).
The BunkerWeb server’s IP is 192.168.1.155, which I’ve also added to the trusted_proxies array, but this did not resolve the issue.

Despite following the standard setup, it seems like BunkerWeb isn’t correctly passing requests to Nextcloud, or the SSL handshake fails due to certificate trust issues.

I created a Cloudflare Origin Certificate and uploaded both the .key and .pem files to BunkerWeb's global configuration. Checked that BunkerWeb is correctly configured to use the Cloudflare Origin Certificate, which is currently being displayed when accessing Nextcloud instance via the domain.

![image](https://github.com/user-attachments/assets/af65d965-105a-4412-a1e4-ef5a0ace99f2)
![image](https://github.com/user-attachments/assets/82bac680-5bb9-468a-b4b1-053e52b783e4)
<img width="1184" alt="image" src="https://github.com/user-attachments/assets/fa255dec-b457-4add-8dd1-21b270602d4a">

**Comment by szaimen** [comment, marked as answer]
Hi, can you follow https://github.com/nextcloud/all-in-one/blob/main/reverse-proxy.md#6-how-to-debug-things?

**Reply by psychedelicu** [reply]
i followed. the solution was to switch back to NPM. Maybe you add a setup for Nextcloud AIO + Bunkerweb in the reverse proxy page.

**Reply by szaimen** [reply]
> Bunkerweb in the reverse proxy page.

I personally do not know Bunkerweb so PRs are welcome: https://github.com/nextcloud/all-in-one/blob/main/reverse-proxy.md#others
